src/index.js

```diff
@@ -37,15 +43,58 @@ window.addEventListener('load', (event) => {
       </div>
     `;
   });
-
   // Add all the divs to the HTML
-  document.querySelector('#memory-board').innerHTML = html;
+  /** 
+   * Understand the game - done
+   * Flip a card
+   * ---Add clickevent listener on card - done
+   * ---Add the classname turned to the class card in the car - done
+   * Flip only two cards -- done
+   * Compare the two cards -- done
+   * ---If card1 and card2 are not the same flip the cards -- done
+   * ---if Card1 and card2 are the same keep them open -- done
+   *  
+      
+   **/
+
+  document.querySelector("#memory-board").innerHTML = html;
+
+  let cardsFlipped = 0;
+  const cardLimit = 2;
+  let firstFlippedCard;
+  let secondFlippedCard;
```

Comment:
You have 4 variables for something that can be done with one array that tracks the cards a user clicks. Replace this with that.